keyboards/jels/jels88/jels88.c

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2021 Joah Nelson (Jels)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation,   either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// not much here ... just indicator LEDs
+
+#include "jels88.h"
+
+// LED indicator pins
+#define CAPS_LED D5
+#define SCROLL_LED D4
+
+void keyboard_pre_init_kb() {
+    setPinOutput(CAPS_LED);
+    setPinOutput(SCROLL_LED);
+    keyboard_pre_init_user();
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if (res) {
+        writePin(CAPS_LED, led_state.caps_lock);
+        writePin(SCROLL_LED, led_state.scroll_lock);
+    }
+    return res;
+}
```

Comment:
Can be replaced with: https://docs.qmk.fm/#/feature_led_indicators

Reply:
I prefer using this method, is nice if people want to change the functionality of these pins later on.

Reply:
It's still possible to do that.

Reply:
More what I mean by that is that using this method, It makes it a little bit simpler changing the functionality of these pins.